### PR TITLE
Setup Survey project

### LINKED ISSUE #130: Add new react app for survey tool (subfolder /surveys)
relates to the Simple Survey Tool #94 discussion

### PR DESCRIPTION
**Pull request by renestalder**
* Add basic vite + react + typescript project for the surveys
* Add useful make commands to use it
* Ensure docker compose runs the ui and survey commands in the correct folders

Resolves #130 

Out of scope:

- GitHub Actions
- Preview deploys
- Firebase setup

### CODE REVIEW

**Review by ssandino — APPROVED** [review]

File: docker-compose.yaml [hunk 1]
```diff
@@ -51,4 +62,5 @@ services:
       - ./:/home/node
     ports:
       - "6006:6006"
+    working_dir: /home/node
```
Comment (mkue) [inline docker-compose.yaml:65]: nit: This is not required anymore. The Dockerfile has `WORKDIR` set to `/home/node`

File: package.json [hunk 2]
```diff
@@ -19,6 +20,8 @@
 		"functions:test": "$npm_execpath --workspace=@socialincome/functions run test",
 		"firebase:serve": "firebase emulators:start --project demo-social-income-prod --config firebase.json --import ./seed",
 		"firebase:export": "firebase emulators:export --project demo-social-income-prod --force ./seed",
+		"survey:build": "$npm_execpath --workspace=@socialincome/ui --workspace=@socialincome/survey run build",
```
Comment (mkue) [inline package.json:23]: Cool! That's really nice that you can build both👍

**Review by mkue — APPROVED** [review]
Very very nice! 🙌  I really really like this setup. It's very clean. And everything works like a charm for me locally. I think I'll switch in the admin project to vite as well, so that we use the same setup in all of our subprojects.

FYI, I added the hosting site configuration to the `.firebaserc` file. This will be required for the deployment.

**Review by ssandino — APPROVED** [review]

File: docker-compose.yaml (hunk 1 above)
Comment (renestalder) [reply docker-compose.yaml:65]: Aaaah, I see. I probably should have rebuilt the image locally, didn't come to my mind.

**Comment by renestalder** [comment]
> Very very nice! raised_hands I really really like this setup. It's very clean. And everything works like a charm for me locally. I think I'll switch in the admin project to vite as well, so that we use the same setup in all of our subprojects.

@mkue 

It's btw. that easy:

```
npm create vite@latest my-vue-app -- --template react-ts
```
See https://vitejs.dev/guide/

**Comment by ssandino** [comment]
@Anup11dwivedi the survey tool setup works now. Everything which is deployed will show up on the webpage [survey.socialincome.org](https://survey.socialincome.org/)

**Pull request merged**